web/src/fileview/fileview.js

```diff
@@ -28,6 +32,74 @@ function doSearch(event, query, newTab) {
   }
 }
 
+function isInCodeScope(selection) {
+  return selection && $(selection.baseNode.parentElement).closest("code").length;
```

Comment:
Tiny, tiny nit: could you end with `.length > 0;` to make explicit that this returns a Boolean?

Reply:
done